modules/cluster-logging-deploy-clo-cli.adoc

```diff
@@ -0,0 +1,266 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploy.adoc
+
+[id="cluster-logging-deploy-clo-cli_{context}"]
+= Install the Cluster Logging Operator using the CLI
+
+You can use the {product-title} CLI to install the Cluster Operator. The Cluster Logging Operator creates and manages the components of the logging stack.
+
+Use the following steps to install the Cluster Logging Operator using the CLI.
+
+.Procedure
+
+. Create a Namespace for the Cluster Logging Operator:
+
+.. Create a Namespace object YAML file (for example, `clo-namespace.yaml`) for the Cluster Logging Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true"
+----
+
+.. Run the following command to create the Namespace:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+----
+$ oc create -f clo-namespace.yaml
+----
+
+. Install the Cluster Logging Operator by creating the following objects:
+
+.. Create an Operator Group object YAML file (for example, `clo-og.yaml`) for the Cluster Logging Operator:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging <1>
+spec:
+  targetNamespaces:
+  - openshift-logging <1>
+----
+<1> You must specify the `openshift-logging` namespace.
+
+.. Create the Operator Group object:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+----
+$ oc create -f clo-og.yaml
+----
+
+.. Create a Subscription object YAML file (for example, `clo-sub.yaml`) to
+subscribe a Namespace to an Operator.
++
```

Comment:
Actually the pre-install of the ElasticsearchOperator is mandatory. I get this error in the logs if i continue past this step w/o having previously installed the ElasticsearchOperator:
```
{"level":"error","ts":1581701722.7534933,"logger":"kubebuilder.controller","msg":"Reconciler error","controller":"clusterlogging-controller","request":"openshift-logging/instance","error":"Unable to create or update logstore for \"instance\": Failure creating Elasticsearch CR: no matches for kind \"Elasticsearch\" in version \"logging.openshift.io/v1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/openshift/cluster-logging-operator/_output/src/github.com/go-logr/zapr/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/src/github.com/openshift/cluster-logging-operator/_output/src/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:217\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/go/src/github.com/openshift/cluster-logging-operator/_output/src/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:158\nk8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/src/github.com/openshift/cluster-logging-operator/_output/src/k8s.io/apimachinery/pkg/util/wait/wait.go:133\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/src/github.com/openshift/cluster-logging-operator/_output/src/k8s.io/apimachinery/pkg/util/wait/wait.go:134\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/src/github.com/openshift/cluster-logging-operator/_output/src/k8s.io/apimachinery/pkg/util/wait/wait.go:88"}
```
This is obvisously correct from an err perspective, because we installed an operator depending on a CR which is provided by the ElasticsearchOperator. I need to delete the clo pod on `openshift-logging` to let it sync the missing CR if I install the ElasticsearchOperator after CLO.